kolibri/plugins/coach/assets/src/views/common/AverageScoreTooltip.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+
+  <div ref="icon" style="display:inline;">
+    <KIcon
+      icon="error"
+      :color="$themeTokens.primary"
+      style="position: relative; top:4px; left: 4px;"
+    />
+    <KTooltip
+      reference="icon"
+      :refs="$refs"
+    >
+      {{ $tr('visibleToLearnersTooltipMessage') }}
+    </KTooltip>
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'AverageScoreTooltip',
+    $trs: {
+      visibleToLearnersTooltipMessage: {
+        message: 'Calculated only from quizzes that were completed',
```

Comment:
Suggest to change the wording here to `Calculated only from quizzes labeled as 'Completed'`. 

Since we are trying to avoid introducing a new status, in my mind this could potentially clarify the concept that even the quizzes which are _started_, but not **submitted** by learners themselves (that is, learners did not actively and intentionally _completed_ them), get **labeled** as _Completed_ once the coach closes the quiz.

Reply:
I think this change might make things harder to understand just because it's a more complicated sentence.

Recommend simply: "Calculated only from completed quizzes". This should be clear that it just refers to the label on the quiz, and not the learner's activity.

Reply:
@radinamatic I'm okay with this - shall we get more input from @jtamiace @lauradanforth ?

Reply:
@nucleogenesis Agreed with @indirectlylit above, could we change to `Calculated only from completed quizzes`?